sherpa/astro/ui/utils.py

```diff
@@ -5105,16 +5105,19 @@ def save_data(self, id, filename=None, bkg_id=None, ascii=True, clobber=False):
             d = self.get_bkg(id, bkg_id)
 
         try:
-            sherpa.astro.io.write_pha(filename, d, ascii, clobber)
+            sherpa.astro.io.write_pha(filename, d, ascii=ascii,
+                                      clobber=clobber)
```

Comment:
They do, but I've taken to exxplicitly labelling optional arguments (since we've had this problem of optional-argument-in-the-wrong-place before), even when the names match like they do here in case the routine is ever changed in the future. This is unlikely here, but I prefer the redundancy given our code base.